### PR TITLE
Create Trang-Doan.md

### PR DESCRIPTION
**Pull request by trangdoan982**

### CODE REVIEW

File: Trang-Doan.md [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Hey, this is Trang Doan.
+My middle name is Creativity.
```
Comment (jasenlo123) [inline Trang-Doan.md]: huh. that's cool. But where's the fancy markdown though?

**Review by jasenlo123 — CHANGES_REQUESTED** [review]
No fancy markdown. Please add a bold or italic!

File: Trang-Doan.md (hunk 1 above)
Comment (trangdoan982) [reply Trang-Doan.md]: fixing

**Review by jasenlo123 — APPROVED** [review]
coolio.

**Pull request merged**